pyproject.toml

```diff
@@ -208,6 +208,7 @@ hiredis = { version = "*", optional = true }
 Pympler = { version = "*", optional = true }
 parameterized = { version = ">=0.7.4", optional = true }
 idna = { version = ">=2.5", optional = true }
+pyicu = { version = ">=2.10.2", optional = true }
```

Comment:
Note that this is the latest version of the package, which can be a point of friction for downstream packagers.

Reply:
I think I might have just used whatever I was using at the time I implemented this change (there doesn't seem to be any recent fix to packaging or typing on that package which would justify otherwise) so it should be fine to relax this constraint if necessary.